src/solvers/dg_curved/dg_3d.jl

```diff
@@ -109,13 +116,15 @@ function calc_interface_flux!(u::AbstractArray{<:Any,5}, mesh::CurvedMesh{3},
 end
 
 
-@inline function calc_interface_flux!(surface_flux_values, left_element, right_element, orientation, u, 
-                                      mesh::CurvedMesh{3}, equations, dg::DG, cache)
+@inline function calc_interface_flux!(surface_flux_values, left_element, right_element,
+                                      orientation, u,
+                                      mesh::CurvedMesh{3}, equations,
+                                      dg::DG, cache)
```

Comment:
Is there a convention how to split lines like this?
It seems like you had to change this everywhere in our code.
We would've tried to make the line lengths as similar as possible, for example:
```suggestion
@inline function calc_interface_flux!(surface_flux_values, left_element, right_element,
                                      orientation, u, mesh::CurvedMesh{3}, 
                                      equations, dg::DG, cache)
```

Reply:
Making the line lengths similar is usually a good approach and I wouldn't say that your version before was somehow "wrong" in any sense. It was just much more convenient for me to shorten some lines a bit when working with multiple open files next to each other (which was helpful to keep consistency between different files).
A rough rule of thumb that I often use is
- not to split arguments belonging to each other such as `nonconservative_terms, equations`
- to put arguments important for dispatch often at the beginning of lines to stress their importance